app/medInria/medLogger.cpp

```diff
@@ -0,0 +1,122 @@
+#include "medLogger.h"
+
+#include <dtkCore>
+#include <medApplication.h>
+#include <medPluginManager.h>
+
+medLogger* medLogger::singleton;
+bool medLogger::logAccessFlag = false;
```

Comment:
It has the same goal though: prevent log messages being written under certain circumstances:
- recursive call into the logger (i.e. dtkLogger generating its own log messages)
- logger has been stopped (to avoid crashes from messages being sent to logger afterwards)